$NetBSD: patch-progman.ini,v 1.1 2021/11/19 18:59:09 sjmulder Exp $

- Comment out defaults, some of which override built in high DPI logic.

--- progman.ini.orig	2021-11-19 11:40:19.553382841 +0000
+++ progman.ini
@@ -6,33 +6,33 @@
 #
 
 [progman]
-font = Microsoft Sans Serif:bold:size=14
-iconfont = Microsoft Sans Serif:size=11
+#font = Microsoft Sans Serif:bold:size=14
+#iconfont = Microsoft Sans Serif:size=11
 
 # Focused windows
-fgcolor = white
-bgcolor = #0000a8
+#fgcolor = white
+#bgcolor = #0000a8
 
 # Unfocused windows
-unfocused_fgcolor = black
-unfocused_bgcolor = white
+#unfocused_fgcolor = black
+#unfocused_bgcolor = white
 
 # Borders
-border_fgcolor = black
-border_bgcolor = #c0c7c8
-border_width = 6
-button_bgcolor = #c0c7c8
-title_padding = 6
+#border_fgcolor = black
+#border_bgcolor = #c0c7c8
+#border_width = 6
+#button_bgcolor = #c0c7c8
+#title_padding = 6
 
 # Launcher
-launcher_fgcolor = black
-launcher_bgcolor = #c0c7c8
+#launcher_fgcolor = black
+#launcher_bgcolor = #c0c7c8
 
 # When not specified, the root color is not changed
 #root_bgcolor = #c0c7c8
 
 # When moving windows, how hard to resist going off-screen
-edgeresist = 80
+#edgeresist = 80
 
 # Custom key bindings can be specified as "Modifier+Key = action".
 [keyboard]
